src/main/java/com/hibernate/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package com.hibernate.model.dto.response;
+
+public class UserResponseDto {
+    private String email;
+    private String password;
```

Comment:
see checklist - we should not expose the secure data, but should return id in all ResponseDtos